src/garage/torch/algos/loss_function_utils.py

```diff
@@ -1,10 +1,25 @@
-"""loss function utilities."""
+"""Differentiable utils typically used when computing loss functions."""
 import torch
 import torch.nn.functional as F
 
 
 def compute_advantages(discount, gae_lambda, max_len, baselines, rewards):
-    """Calculate advantages."""
+    """
+    Calculate advantages.
+
+    Advantages are a discounted cumulative sum.
+
+    Args:
+        discount (float): Discount.
+        gae_lambda (float): Lambda used for generalized advantage estimation.
+        max_len (int): Maximum length of a single rollout.
+        baselines (torch.Tensor): The baseline values.
+        rewards (torch.Tensor): The rewards.
+
+    Returns:
+        advantages (torch.Tensor): advantage values
```

Comment:
Return values in docstrings are not named. This would be documented:

```python
Returns:
    torch.Tensor: Advantage values, as calculated using Generalized Advantage Estimation
```

Reply:
fixed